build_tooling/mongod.sh

```diff
@@ -0,0 +1,5 @@
+tooling_dir="$(dirname $BASH_SOURCE)"
+
+cp $tooling_dir/mongodb.repo /etc/yum/repos.d/
```

Comment:
Build reported

    cp: cannot create regular file ‘/etc/yum/repos.d/’: Not a directory